Guide/Constructive/Bonetrousle/README.md

```diff
@@ -0,0 +1,97 @@
+# Bonetrousle
+[https://www.hackerrank.com/challenges/bonetrousle](https://www.hackerrank.com/challenges/bonetrousle)
+
+Difficulty: Medium
+
+
+## Problem
+### Statement
+Once upon a time, Papyrus the skeleton went to buy some pasta from the store. The store's inventory is bare-bones and they only sell one thing — boxes of uncooked spaghetti! The store always stocks exactly k boxes of pasta, and each box is numbered sequentially from 1 to k. This box number also corresponds to the number of sticks of spaghetti in the box, meaning the first box contains 1 stick, the second box contains 2 sticks, the third box contains 3 sticks, …, and the k<sup>th</sup> box contains k sticks. Because they only stock one box of each kind, the store has a tendon-cy to sell out of spaghetti.
+
+During each trip to the store, Papyrus likes to buy exactly n sticks of spaghetti by purchasing exactly b boxes (no more, no less). Not sure which boxes to purchase, Papyrus calls Sherlock Bones for help but he's also stumped! Do you have the guts to solve this puzzle?
+
+Given the values of n, k, and b for t trips to the store, determine which boxes Papyrus must purchase during each trip. For each trip, print a single line of b distinct space-separated integers denoting the box number for each box of spaghetti Papyrus purchases (recall that the store only has one box of each kind). If it's not possible to buy n sticks of spaghetti by purchasing  boxes, print -1 instead.
+
+###Input Format
+
+The first line contains a single integer, t, denoting the number of trips to the store.
+Each of the t subsequent lines describes a trip to the store in the form of three space-separated integers describing the respective values of n (the number of sticks to buy), k (the number of boxes the store has for sale), and b (the number of boxes to buy) for that trip to the store.
+
+###Constraints
+
+* 1 <= t <= 20
```

Comment:
Don't worry about this, we are assuming the user is going to actually read the problem. Get rid of the "###Constraints" section.